girder/models/setting.py

```diff
@@ -215,6 +215,22 @@ def validateCoreUserDefaultFolders(self, doc):
                 'User default folders must be either "public_private" or '
                 '"none".', 'value')
 
+    def validateCoreRouteTable(self, doc):
+        nonEmptyRoutes = [route for route in doc['value'].values() if route]
+        if 'girder' not in doc['value'] or not doc['value']['girder']:
+            raise ValidationException('Girder must be routeable.')
```

Comment:
Something more expansive here, perhaps

    Girder must have a valid webroot and route